.github/workflows/terraform-plan-apply.yml

```diff
@@ -21,12 +26,9 @@ jobs:
           set -o pipefail
           set -o nounset
           # set -o xtrace
-
```

Comment:
Why did you delete all empty lines?

Reply:
Mistake caused by auto reformat on VS code... fixed